releasewarrior/templates/firefox/release-rc.json.tmpl

```diff
@@ -31,7 +31,7 @@
             {
                 "alias": "wnp",
                 "description": "setup whatsnew page",
-                "docs": "https://wiki.mozilla.org/Release:Release_Automation_on_Mercurial:Updates_through_Shipping#Set-up_whatsnew_page",
+                "docs": "https://github.com/mozilla-releng/releasewarrior-2.0/blob/master/docs/release-promotion/desktop/howto-rc.md",
```

Comment:
Should we update the non RC template too? AKA dot releases: https://github.com/mozilla-releng/releasewarrior-2.0/search?utf8=%E2%9C%93&q=Set-up_whatsnew_page&type=

^ Looks like there is mention of the old whatsnewpage wiki in the "manual partial gen" doc. I suppose this applies there too? If the old schema (wiki.m.o wnp gen docs) still work, I suppose it's not as critical to update.

Reply:
I didn't realize there was another reference there :). I'll update it.